### ISSUE TITLE
nice project, however you have some misleading information in your README

### ISSUE THREAD

**Issue by vpereira** [alanfranz/docker-rpm-builder#25]
> hosted build systems like copr or OBS are an hassle because you can't try the builds on your own machine, and this makes the packaging process painfully slow,

I don't know if copr can do it too, but with OBS you can build everything locally. It's even recommended to do so. Please check the documentation for more information:
https://en.opensuse.org/openSUSE:Build_Service_Tutorial#Perform_the_local_build

happy hacking!

**Comment by aframachines** [comment]
Thank you.. I'll check that out!
Il 25/apr/2016 04:04 PM, "Victor Pereira" notifications@github.com ha
scritto:

**Comment by alanfranz** [comment]
Hello @vpereira , I checked the doc from the wiki that you linked. It was a long time since I tried OBS.

As far as I can understand, I can execute just the **build step** on my own machine. That step is performed by fetching the source and the configuration from the remote OBS server. If I want to change something, I must perform something like "change source/config -> push modifications -> execute build".

I admit that the wording is not good, by the way. Strictly speaking, I CAN try copr builds on my own machine (via mock) even though the setup isn't trivial. I'll change the way I express that concept.
